src/ros_commo.py

```diff
@@ -444,10 +445,8 @@ def __init__(self):
 			self.language_perceived_text_cb)
 
 		# Chat infrastructure text.
-		# from chatbot.msg import ChatMessage
-		#rospy.Subscriber("/robot/speech", chatbot/ChatMessage,
-		#rospy.Subscriber("chatbot_speech", chatbot/ChatMessage,
-		#	self.chat_perceived_text_cb)
+		rospy.Subscriber("chatbot_speech", ChatMessage,
```

Comment:
The point is that HEAD still does not work, and is not in a usable state.  I don't have the time to debug it and figure out what is broken; I want to move on and work on other things.